algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableTrialFactory.java

```diff
@@ -98,8 +98,9 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
                                     .evidenceLevelDetails(EvidenceLevelDetails.CLINICAL_STUDY)
                                     .direction(EvidenceDirection.RESPONSIVE)
                                     .evidenceUrls(Sets.newHashSet("https://clinicaltrials.gov/study/" + trial.nctId()))
-                                    .date(LocalDate.EPOCH)
-                                    .description(Strings.EMPTY)
+                                    .entryDate(LocalDate.EPOCH)
+                                    .efficacyDescription(Strings.EMPTY)
+                                    .efficacyDescriptionYear(Strings.EMPTY)
```

Comment:
Some thoughts here:
 - Would suggest to set the fields in the order of the datamodel
 - Why not use the actual entry date here?
 - If you always set the efficacy fields to Empty, maybe they belong in `Treatment` rather than `ActionableEvent`?